jenkinsfile-runner-steward-image/scripts/run.sh

```diff
@@ -162,6 +162,35 @@ function with_termination_log() {
   return "$rc"
 }
 
+function with_retries() {
+  local retry_interval=$1 timeout_seconds=$2
+  local cmd=("${@:3}")
+  local rc
+
+  # Use Bash built-in variable SECONDS
+  start=$SECONDS
```

Comment:
I tested it yesterday. It should be EPOCHSECONDS instead of SECONDS. I will also fix my code.

Reply:
And the variable should be local, not global.